shim/src/realm.js

```diff
@@ -95,48 +72,43 @@ function createRealmRec(unsafeRec) {
   return realmRec;
 }
 
-function initRootRealm(selfClass, self, options) {
-  options = Object(options); // Todo: sanitize
-  // note: 'self' is the instance of the Realm, and 'selfClass' is the
-  // Realm constructor (facade) we build in buildChildRealm().
+function initRootRealm(parentUnsafeRec, self, options) {
+  options = Object(options); // todo: sanitize
 
   // In 'undefined' mode, intrinics are not provided, we create a root
```

Comment:
This comment looks stale even before this change. IIUC, it was for when we had common creation logic for root realms and compartments, which went away a long time ago.